### DISCUSSION TITLE
Keine Verbindung zum Server bei eigener Installation

### DISCUSSION THREAD

**Discussion by stweil** [UB-Mannheim/escriptorium#11] [Q&A] [answered]
Frage eines Anwenders:
Ich habe eScriptorium gemäß der Anleitung installiert, jedoch kann ich meine Instanz unter http://localhost:8000/ nicht aufrufen, da keine Verbindung zum Server aufgebaut werden kann.

**Comment by lawi21** [comment, marked as answer]
Dass Sie die URL http://localhost:8000/ aufrufen können, setzt voraus, dass der Server läuft. Hierfür muss der Befehl `python manage.py runserver --settings escriptorium.local_settings` erfolgreich ausgeführt worden sein. Wiederholen Sie den Schritt einfach nochmals und achten Sie auf eventuelle Fehlermeldungen.
